### PR TITLE
Feature/sdk bump

### PR DESCRIPTION
**Pull request by bowtiedpyro**
Merge sdk_bump into release/redesign - latest version of SDK, updated functions and structure. UI updates, market data update

### CODE REVIEW

File: contexts/AccountsProvider.tsx
Comment (BowTiedHeron) [inline contexts/AccountsProvider.tsx:136]: need to be expanded or deprecated.

File: helpers/loanHelpers/userCollection.tsx
Comment (BowTiedHeron) [inline helpers/loanHelpers/userCollection.tsx:75]: still needs to be exported? @bowtiedpyro

File: helpers/loanHelpers/userCollection.tsx
Comment (BowTiedHeron) [inline helpers/loanHelpers/userCollection.tsx:83]: also needs to be exported?

File: helpers/loanHelpers/userCollection.tsx
Comment (BowTiedHeron) [inline helpers/loanHelpers/userCollection.tsx:123]: is this relevant? dummy chart data?

File: helpers/loanHelpers/userCollection.tsx
Comment (BowTiedHeron) [inline helpers/loanHelpers/userCollection.tsx:179]: @bowtiedpyro verify if this function can be removed (as discussed)

File: helpers/loanHelpers/userCollection.tsx
Comment (BowTiedHeron) [inline helpers/loanHelpers/userCollection.tsx:204]: was this *100 migrated from sdk to frontend?
verify if it wasnt correct at first/if its correct now

File: helpers/loanHelpers/userCollection.tsx
Comment (BowTiedHeron) [inline helpers/loanHelpers/userCollection.tsx:356]: move BN to sdk?

File: helpers/marketHelpers/index.tsx
Comment (BowTiedHeron) [inline helpers/marketHelpers/index.tsx:293]: double

**Review by BowTiedHeron — CHANGES_REQUESTED** [review]
need to finish reviewing borrow, lend and liquidations tomorrow morning, as well as walking through comments.

File: pages/borrow/index.tsx
Comment (BowTiedHeron) [inline pages/borrow/index.tsx:210]: rename constant to reserve price or something related to SPL tokens.
first it was SOL, now not anymore.

File: pages/borrow/index.tsx
Comment (BowTiedHeron) [inline pages/borrow/index.tsx:221]: rename solValue

File: pages/borrow/index.tsx
Comment (BowTiedHeron) [inline pages/borrow/index.tsx:239]: @bowtiedpyro create marketID array based on loop through market collections.
don't want to keep track of markets at multiple places, but reuse values.

File: pages/borrow/index.tsx
Comment (BowTiedHeron) [inline pages/borrow/index.tsx:286]: consider renaming mData to something more self explanatory/obvious (keeping 3rd parties in mind)

File: pages/borrow/index.tsx
Comment (BowTiedHeron) [inline pages/borrow/index.tsx:312]: still BN or string? is conversion to Number required?

File: pages/borrow/index.tsx
Comment (BowTiedHeron) [inline pages/borrow/index.tsx:1022]: @bowtiedpyro update to value used in SSR branch as discussed
make sure that all related updates regarding the SDK bump are included in this PR/branch 
(no additional ones in the SSR branch)

File: pages/lend/index.tsx
Comment (BowTiedHeron) [inline pages/lend/index.tsx:173]: rename func as mentioned for borrow page

File: pages/lend/index.tsx
Comment (BowTiedHeron) [inline pages/lend/index.tsx:191]: only sol?

File: pages/lend/index.tsx
Comment (BowTiedHeron) [inline pages/lend/index.tsx:258]: only sol?

File: pages/lend/index.tsx
Comment (BowTiedHeron) [inline pages/lend/index.tsx:401]: ideally we dont use fixed timeouts, but based on loading state

File: pages/liquidate/index.tsx
Comment (BowTiedHeron) [inline pages/liquidate/index.tsx:252]: rename solValue

File: pages/liquidate/index.tsx
Comment (BowTiedHeron) [inline pages/liquidate/index.tsx:299]: add wait for confirmation? @bowtiedpyro

File: pages/liquidate/index.tsx
Comment (BowTiedHeron) [inline pages/liquidate/index.tsx:322]: remove tx outcome

File: pages/liquidate/index.tsx
Comment (BowTiedHeron) [inline pages/liquidate/index.tsx:302]: remove tx outcome

File: pages/liquidate/index.tsx
Comment (BowTiedHeron) [inline pages/liquidate/index.tsx:327]: @bowtiedpyro sync with @yuriagile if this should be Number or BN

**Review by BowTiedHeron — CHANGES_REQUESTED** [review]
added final comments that need to be reviewed before merge
will sync again before merging!

File: contexts/AccountsProvider.tsx
Comment (bowtiedpyro) [reply contexts/AccountsProvider.tsx:136]: Done.

File: helpers/loanHelpers/userCollection.tsx
Comment (bowtiedpyro) [reply helpers/loanHelpers/userCollection.tsx:75]: For now yes, will check with sdk / open request for shipping it there.

File: helpers/loanHelpers/userCollection.tsx
Comment (bowtiedpyro) [reply helpers/loanHelpers/userCollection.tsx:123]: Is required for future work - we should leave it there for now.

File: helpers/loanHelpers/userCollection.tsx
Comment (bowtiedpyro) [reply helpers/loanHelpers/userCollection.tsx:179]: This formats our obligations as presented in the FED, for now we need this. Will bring it up with SDK dev to validate if we want to ship this logic to the SDK

File: helpers/loanHelpers/userCollection.tsx
Comment (bowtiedpyro) [reply helpers/loanHelpers/userCollection.tsx:356]: doesn't need to be BN - converted it back, simply is a calculation now - but will discuss this to ship to SDK

File: helpers/loanHelpers/userCollection.tsx
Comment (bowtiedpyro) [reply helpers/loanHelpers/userCollection.tsx:204]: it's the forumla for the risk factor, its the total sum of debt - div. by the length of the array div. by the nft price, which gives us a 00,XX% value - so we multiply by 100 to get a XX,XX% value

File: pages/borrow/index.tsx
Comment (bowtiedpyro) [reply pages/borrow/index.tsx:239]: fixed, currently not in use - works: future work we can implement this.

File: helpers/marketHelpers/index.tsx
Comment (bowtiedpyro) [reply helpers/marketHelpers/index.tsx:293]: removed

File: pages/borrow/index.tsx
Comment (bowtiedpyro) [reply pages/borrow/index.tsx:210]: done

File: pages/borrow/index.tsx
Comment (bowtiedpyro) [reply pages/borrow/index.tsx:286]: nice to have

File: pages/borrow/index.tsx
Comment (bowtiedpyro) [reply pages/borrow/index.tsx:312]: fixed, bn was not needed, nor number

File: pages/borrow/index.tsx
Comment (bowtiedpyro) [reply pages/borrow/index.tsx:1022]: done

File: pages/lend/index.tsx
Comment (bowtiedpyro) [reply pages/lend/index.tsx:173]: done

File: pages/lend/index.tsx
Comment (bowtiedpyro) [reply pages/lend/index.tsx:258]: done

File: pages/lend/index.tsx
Comment (bowtiedpyro) [reply pages/lend/index.tsx:191]: done

File: pages/lend/index.tsx
Comment (bowtiedpyro) [reply pages/lend/index.tsx:401]: true, for now we leave it like this - need to see where the delay is

File: pages/liquidate/index.tsx
Comment (bowtiedpyro) [reply pages/liquidate/index.tsx:252]: done

File: pages/liquidate/index.tsx
Comment (bowtiedpyro) [reply pages/liquidate/index.tsx:322]: done

File: pages/liquidate/index.tsx
Comment (bowtiedpyro) [reply pages/liquidate/index.tsx:302]: done

File: pages/liquidate/index.tsx
Comment (bowtiedpyro) [reply pages/liquidate/index.tsx:327]: will need to validate with Yuri, for now we can leave it like this

File: pages/liquidate/index.tsx
Comment (bowtiedpyro) [reply pages/liquidate/index.tsx:299]: Will test this out

**Pull request closed without merging**
